daemon/daemon_unix.go

```diff
@@ -906,9 +906,9 @@ func (daemon *Daemon) initNetworkController(config *config.Config, activeSandbox
 
 	// Initialize default network on "host"
 	if n, _ := controller.NetworkByName("host"); n == nil {
-		if _, err := controller.NewNetwork("host", "host", "", libnetwork.NetworkOptionPersist(true)); err != nil {
-			return nil, fmt.Errorf("Error creating default \"host\" network: %v", err)
-		}
+		// if _, err := controller.NewNetwork("host", "host", "", libnetwork.NetworkOptionPersist(true)); err != nil {
+		// 	return nil, fmt.Errorf("Error creating default \"host\" network: %v", err)
+		// }
```

Comment:
let's confirm this again. if this is a runtime issue, let's use `GOOS=darwin` workaround.

Reply:
Yup, that worked as of now. Pushed.